dist/overwolf/manifest.json

```diff
@@ -13,8 +13,11 @@
     "launcher_icon": "icons/desktopIcon.ico",
     "window_icon": "icons/taskbarIcon.png"
   },
-  "permissions": ["VideoCaptureSettings"],
+  "permissions": ["VideoCaptureSettings", "Extensions", "Streaming", "Profile", "GameInfo"],
   "data": {
+    "block_top_window_navigation": true,
+    "popup_blocker": true,
+    "mute": true,
```

Comment:
These settings are correct, but in the wrong place.
https://overwolf.github.io/docs/api/manifest-json

It should be under data->windows->desktop (the Window(s) where you like to display ads).